src/components/ShelterListItem/ShelterListItem.tsx

```diff
@@ -22,13 +22,20 @@ const ShelterListItem = (props: IShelterListItemProps) => {
     );
 
   const tags = useMemo(
-    () =>
-      data.supplies
-        .filter((s) => s.priority >= SupplyPriority.Needing)
-        .sort((a, b) => b.priority - a.priority),
+    () => {
+      return data.supplies
+        .filter((s) => (s.priority >= SupplyPriority.Needing || s.name.toLowerCase().includes(getWordToFilterVolunteer())))
```

Comment:
Temos uma categoria chamada "Voluntariado" e uma chamada "Especialistas e Profissionais". Fazer o filtro pelo nome da categoria ao invés do nome do item permite adicionar outros tipos de voluntário independente do nome, por exemplo, Médicos e Veterinários.

Reply:
beleza, quando eu tava olhando o retorno do BE náo tinha essa propriedade, vou alterar o BE para filtrar por isso